src/helpers/cli.ts

```diff
@@ -0,0 +1,11 @@
+import { Command, Interfaces, Flags, Help } from "@oclif/core";
+
+export const helpFlag = (opts: Partial<Interfaces.BooleanFlag<boolean>> = {}) => {
+  return Flags.help({
+    ...opts,
+    parse: async (_, cmd) => {
+      new Help(cmd.config).showCommandHelp(cmd.constructor as Command.Class);
+      return cmd.exit(0) as never;
```

Comment:
Can you clarify the purpose of this helper and the parse callback?

Reply:
Yup, so, like i mentioned in the description, this is our custom help flag. The parse callback is the same as the [builtin oclif help parse callback](https://hub/oclif/core/blob/ff76400fb0bdfc4be0fa93056e86183b9205b323/src/flags.ts#L112), just  with `showCommandHelp` instead of `showHelp`, since `showHelp` relies on registering all of the commands internally with oclif, and then fetching the command with `Config.findCommand`. That's more ceremony than we need with our single command cli, so we just bypass that and go directly to `showCommandHelp`.